src/core/shared_state/currentMapPosition.ts

```
@@ -23,50 +24,56 @@
+        if (mapCenter.lng !== lng || mapCenter.lat !== lat || mapZoom !== zoom) {
+          requestAnimationFrame(() => {
+            map?.jumpTo({
+              center: [lng, lat],
+              zoom: zoom,
+            });
+          });
         }
-      }
+      }, 100);
+
+      state = position;
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)